package.json

```diff
@@ -22,16 +22,16 @@
         "changelog": "git cliff v0.0.4..$(git describe --tags $(git rev-list --tags --max-count=1)) --output CHANGELOG.md"
     },
     "dependencies": {
-        "@codemirror/basic-setup": "^0.19.0",
-        "@codemirror/highlight": "^0.19.6",
-        "@codemirror/lang-javascript": "^0.19.3",
-        "@codemirror/lang-json": "^0.19.1",
-        "@codemirror/language": "^0.19.7",
-        "@codemirror/legacy-modes": "^0.19.0",
-        "@codemirror/search": "^0.19.4",
-        "@codemirror/state": "^0.19.6",
-        "@codemirror/stream-parser": "^0.19.2",
-        "@codemirror/view": "^0.19.28",
+        "@codemirror/basic-setup": "^0.20.0",
+        "@codemirror/highlight": "^0.19.8",
```

Comment:
this dependency is not required anymore as of this release info: https://discuss.codemirror.net/t/release-0-20-0/4302
it was replaced by `"@lezer/highlight": "^0.16.0",` in line 34